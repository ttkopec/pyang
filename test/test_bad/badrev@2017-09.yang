module badrev {
    namespace 'urn:badrev';

    prefix 'badrev';
}
