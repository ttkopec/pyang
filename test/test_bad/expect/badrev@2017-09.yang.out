badrev@2017-09.yang:1: warning: FILENAME_BAD_REVISION
